docs/getting-started-on-snowplow-open-source/setup-snowplow-on-aws/setup-destinations/snowflake/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Snowflake"
+date: "2020-02-26"
+sidebar_position: 0
+---
+
+[Snowflake](https://www.snowflake.com/) is a fully-managed cloud data warehouse with rich support of semi-structured data and tight integration with AWS. Snowplow allows to load data into Snowflake with [Snowplow RDB Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/index.md), which supports loading into Snowflake.
```

Comment:
```suggestion
[Snowflake](https://www.snowflake.com/) is a fully-managed cloud data warehouse with rich support of semi-structured data and tight integration with AWS. Snowplow allows to load data into Snowflake with [Snowplow RDB Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/index.md).
```

Reply:
I think we can provide a tiny bit more guidance here, otherwise the page looks somewhat bare. For example, we can mention that the installation instructions are at the linked page, that the user needs to pick a transformer, and that there is a terraform stack available for the Snowflake RDB loader (I think?). Same comment applies to a copy of this text below. (If it gets much longer, maybe it’s worth extracting it into a reusable fragment.)